COPYING.txt

```diff
@@ -1229,9 +1229,7 @@ The following software used with SINGULAR have their own copyright: the
 omalloc library, the readline library, the Gnu Multiple Precision
 Library (GMP), NTL: A Library for doing Number Theory (NTL), the Multi
 Protocol library (MP), the Singular-Factory library, the
-Singular-libfac library, surfex, and, for the Windows distributions the
```

Comment:
These lines are part of Singular's `COPYING`, we're including the whole thing verbatim.

Reply:
done